components/Tags.tsx

```diff
@@ -1,53 +1,98 @@
-import { useEffect } from "react";
+import { isDefined } from "helpers/tinaHelpers";
+import { ReactElement, useEffect } from "react";
 import { resetPaginationPage } from "redux/slices/pagination";
 import { resetTags, setTags } from "redux/slices/selectedTags";
 import { useAppDispatch, useAppSelector } from "redux/typesHooks";
+import { useEditState } from "tinacms/dist/react";
+
+export interface TagData {
```

Comment:
```suggestion
export interface Tag {
```
`Data` suffix does not add extra information for this interface